src/GUI_draft/templates/step1.html

```diff
@@ -5,10 +5,22 @@
     <h2 class="cell">Scenario Parameters</h2>
 
     <div title="Click here"><h4><a href="/scenario_parameters">Enter Scenario Parameters VC</a></h4></div>
-    <div title="Click here"><h4><a href="/load_proj_params">Load Scenario Parameters ... </a></h4></div>
-
+    <div title="Click here to load the file containing the scenario parameters">
+        <h4>
+            <label for="scenarioParamsFileUpload" class="button alert">Load Scenario Parameters ...</label>
+            <input type="file" id="scenarioParamsFileUpload" class="show-for-sr">
+        </h4>
+    </div>
 
     <p class="cell">Please choose which energy sector you would like to cover in your energy cell/energy system</p>
-    <div class="cell small-4" title="Click here"><h4><a href="/es_sector">Energy System Sector selector VC</a></h4></div>
 
-{% endblock progression_content %}
+    <div  class="sysselecttext">
+        <fieldset class="fieldset">
+            <legend>Sectors in the Energy System</legend>
+                <input id="checkbox12" type="checkbox"><label for="checkbox12">Electricity</label>
```

Comment:
Use a list of energy carrier to fill with a for loop :)

Reply:
I did it very roughly, will do that later on. Actually, should we allow the user to define the sectors? Maybe there are other sectors that they want to simulate?

Reply:
The sector list is bounded by the sector we are able to simulate with oemof --> no user choice there

Reply:
But OEMOF is a framework. You can create your own sector and the components as long as they can modeled as a LP problem, AFAIK

Reply:
Yes, but you still need to be able to create the components and then make sure you can initialize them via the GUI, this is only done by the GUI devs, i.e. us in that case. So if a sector is not implemented it is not because the user wants to add an energy sector that they straightforwardly can, there is some involvement required there and the list of implemented sectors can be updated then. When you mention "user", I take it as "GUI-user" and I don't assume they do have programming skills.

Reply:
got it, thanks for the explanation!